.eslintrc.json

```diff
@@ -19,6 +19,7 @@
     "no-shadow-restricted-names": 2,
     "no-undef-init": 2,
     "no-undefined": 2,
+    "no-use-before-define": 2,
```

Comment:
Didn't this rule cause issues for papa's angular style? Because you need to use a function before you define it, eg when declaring a directive.